app/src/main/java/com/example/tour_a_versity_v3/ui/MapsActivity.kt

```diff
@@ -35,39 +38,61 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWi
         this.map = googleMap
         map.mapType = GoogleMap.MAP_TYPE_HYBRID
         map.setOnInfoWindowClickListener(this)
-        //set values for TUC
-        val tucLat = 39.13175
-        val tucLng = -84.51774
-        val zoomLevel = 17f
-        //create LatLng object for TUC
-        val tucLatLng = LatLng(tucLat, tucLng)
+
         //move camera to TUC & add marker
-        map.moveCamera(CameraUpdateFactory.newLatLngZoom(tucLatLng, zoomLevel))
-        tucMarker = map.addMarker(MarkerOptions()
-                .position(tucLatLng)
-                .title("TUC")
-                .snippet("Tangeman University Center")
-        )
+        moveCameraToTuc()
+        tucMarker = getTucMarker()
 
         //create another marker for CECH
-        val cechLat = 39.130315
-        val cechLng = -84.518680
-        val cechLatLng = LatLng(cechLat, cechLng)
-        cechMarker = map.addMarker(MarkerOptions()
-                .position(cechLatLng)
-                .title("CECH")
-                .snippet("College of Education, Criminal Justice, Human Services, and IT")
-        )
```

Comment:
* I recommend keeping these methods slim, or they might end up being really long because they have to accomplish different tasks. You can abstract out pieces of related code into a private method.
* You are creating markers using data here. Handling data is supposed to be the job of a DTO/ViewModel, not your activity class. Since you don't have a ViewModel yet, I abstracted this out into a private function.